### PR TITLE
Ch setup travis # #167977789

### PR DESCRIPTION
**Pull request by willywax**
#### What does this PR do?
Set Up Travis , Coverall and Code Climate
#### Description of Task to be completed?
Integrated Travis, Coveralls and Code Climate 
Add badges for Travis, Coveralls and Code Climate
#### How should this be manually tested?
Badges should display on develop readme
#### What are the relevant pivotal tracker stories?
[#167977789](https://www.pivotaltracker.com/story/show/167977789)

### CODE REVIEW

**Comment by j0flintking02** [comment]
squash you commit, please

**Review by Bahatiroben — APPROVED** [review]
The tests are ok. It is good. Yet you have bunch of commits. You may squash them into one.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - psql -c "CREATE USER thor WITH PASSWORD 'thanos';" -U postgres
+  - npm run build
+  - npm install -g sequelize -cli
+  - sequelize db:migrate
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build 
+
+script:
+  - yarn test
```
Comment (Dkabiswa) [inline .travis.yml]: let us stick to npm package

**Review by Dkabiswa — CHANGES_REQUESTED** [review]
let us stick to npm

**Review by drayzii — APPROVED** [review]

**Review by Dkabiswa — APPROVED** [review]
LGTM

**Pull request merged**